circle.yml

```diff
@@ -150,6 +150,7 @@ jobs:
           DEBUG: 'true'
           DEFAULT_FORK: 'open-craft/edx-platform'
           LOAD_BALANCER_FRAGMENT_NAME_PREFIX: 'integration-'
+          SIMPLE_THEME_SKELETON_THEME_VERSION: 'samuel/remove-v2'
```

Comment:
@0x29a, this PR branch should be merged to a properly named branch and that should be used here. :) If that branch is `master`, this should be removed. :)

Reply:
@lgp171188, I covered this in `Author notes and concerns` section of the PR description, but I think we have to merge this branch in `master` after merging this PR because doing this _before_ might break customization of non-Koa trial instances.